src/ol/interaction/Draw.js

```diff
@@ -937,7 +936,7 @@ class Draw extends PointerInteraction {
       this.geometryFunction_(this.sketchCoords_, geometry, projection);
     }
 
-    if (coordinates.length === 0) {
+    if (coordinates.length === 1) {
```

Comment:
This one-line change is what I intended. The rest of the changes is to not break tests and to keep the `tracing.html` example work like it used to. Useful changes though, because `appendCoordinates()` can now also be used to start a new drawing.

Reply:
There are several `if (coordinates.length) {` checks in the geometry functions which may no longer be needed as a result of that.

Reply:
Thanks! I knew those checks shouldn't be there.